Report Exporter/Sdl.Community.ReportExporter/Helpers/StudioAnalysisReport.cs

```diff
@@ -3,170 +3,192 @@
 using System.IO;
 using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
 using System.Xml.Linq;
 using Sdl.Community.ReportExporter.Model;
 
 namespace Sdl.Community.ReportExporter.Helpers
 {
 	public class StudioAnalysisReport
 	{
+		public static readonly Log Log = Log.Instance;
+
 		public string ReportFile { get; }
 		public IEnumerable<AnalyzedFile> AnalyzedFiles { get;  }
 
 		public StudioAnalysisReport(string pathToXmlReport)
 		{
-			ReportFile = pathToXmlReport;
-
-			if (!File.Exists(ReportFile))
+			try
 			{
-				throw new FileNotFoundException("Analysis report not found", ReportFile);
-			}
+				ReportFile = pathToXmlReport;
 
-			var xDoc = XDocument.Load(ReportFile);
-			AnalyzedFiles = from f in xDoc.Root.Descendants("file")
-				select new AnalyzedFile(f.Attribute("name").Value, f.Attribute("guid").Value)
+				if (!File.Exists(ReportFile))
 				{
-					Results = from r in f.Element("analyse").Elements()
-						select new BandResult(r.Name.LocalName)
-						{
-							Segments = r.Attribute("segments") != null ? int.Parse(r.Attribute("segments").Value) : 0,
-							Words = r.Attribute("words") != null ? int.Parse(r.Attribute("words").Value) : 0,
-							Characters = r.Attribute("characters") != null ? int.Parse(r.Attribute("characters").Value) : 0,
-							Placeables = r.Attribute("placeables") != null ? int.Parse(r.Attribute("placeables").Value) : 0,
-							Tags = r.Attribute("tags") != null ? int.Parse(r.Attribute("tags").Value) : 0,
-							Min = r.Attribute("min") != null ? int.Parse(r.Attribute("min").Value) : 0,
-							Max = r.Attribute("max") != null ? int.Parse(r.Attribute("max").Value) : 0,
-							FullRecallWords = r.Attribute("fullRecallWords") != null ? int.Parse(r.Attribute("fullRecallWords").Value) : 0,
-						}
-				};
+					throw new FileNotFoundException("Analysis report not found", ReportFile);
```

Comment:
Nu sunt sigur ca e ok aruncatul de exceptii direct din try... pare sa nu fie o exceptie reala ci doar un alt caz din flowul normal. (explicatie mai buna [aici](https://stackoverflow.com/a/3293682) si [aici](https://stackoverflow.com/a/651651)

Reply:
poate un messagebox afisat pt user in loc de exceptie.

Reply:
Voi modifica cu messagebox (care ii definit prin interfata de IMessageBoxService)